game_eggs/urbanterror/egg-urban-terror4-3-4.json

```diff
@@ -0,0 +1,152 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-02-09T19:34:16+00:00",
+    "name": "Urban Terror 4.3.4",
+    "author": "Juli199696@web.de",
+    "description": "Urban Terror started as a realism based \"total conversion\" mod for Quake III Arena. Nowadays, Urban Terror is played using the free, stand-alone, ioquake3 engine modified, which doesn't require Quake III Arena, in addition to being an add-on to id Software's Quake III Arena.",
+    "features": null,
+    "docker_images": {
+        "quay.io\/pterodactyl\/core:glibc": "quay.io\/pterodactyl\/core:glibc"
```

Comment:
This is dead old and should not be used anymore. Please use the debian image.

Reply:
Using current Debian Yolk now from ghcr.io/parkervcp/installers:debian

Reply:
I changed it now to ghcr.io/parkervcp/yolks:debian

I remember replacing it yesterday but it seems i forgot to click in save after that and reinstalled the server. It works now with the debian yolk. Not with the installer debian :)